examples/package.json

```
@@ -21,13 +21,13 @@
     "typescript": "5.3.3"
   },
   "dependencies": {
-    "@asteasolutions/zod-to-openapi": "^6.3.1",
+    "@asteasolutions/zod-to-openapi": "^7.0.0",
     "aws-cdk-lib": "2.117.0",
     "aws-lambda": "^1.0.7",
     "cdk-practical-constructs": "file:../lib/dist/cdk-practical-constructs-0.0.1.tgz",
     "constructs": "^10.3.0",
     "esbuild": "^0.19.11",
     "openapi3-ts": "^4.2.1",
-    "zod": "^3.22.4"
+    "zod": "^3.23.8"
```

```suggestion
    "zod": "~3.23.8"
```
As zod is breaking its interface in minor releases, it's a good idea to stick only to bug fixes updates

---

does it matter? the idea is that we'll use the zod version from the host